### ISSUE TITLE
Multiline arguments

### ISSUE THREAD

**Issue by scyzoryck** [blackfireio/player#33]
Hey. 
Did you think about adding possibility to create multiline arguments in scenarios? It would increase readability of bigger JSON bodies for sending POST / PUT requests. :)

**Comment by romainneutron** [comment]
Hello, this is definitely something we have to address

**Comment by scyzoryck** [comment]
Thanks for information! 
If I can help you to implement it somehow, please let me know!

**Comment by romainneutron** [comment]
Hello, 

this has been implemented in version 0.13
It can be used like this:

```yaml
        scenario
            visit url('/login')
                method 'POST'
                body
                """
                {
                    user: "john",
                    password: "doe"
                }
                """
```
